### PR TITLE
chore(Banner): updated placement in sticky demo

### LINKED ISSUE #10577: Update sticky banner in-page banner example
can we update the [basic sticky banner ](https://staging-v6.patternfly.org/components/banner/react-demos#basic-sticky-banner) demo to have the banner appear above the breadcrumbs? 
image

### PR DESCRIPTION
**Pull request by thatblindgeye**
**What**: Closes #10577 

**Additional issues**:

### CODE REVIEW

**Review by lboehling — APPROVED** [review]
looks great! thank you!!

File: packages/react-core/src/components/Page/Page.tsx [hunk 1]
```diff
@@ -76,6 +76,8 @@ export interface PageProps extends React.HTMLProps<HTMLDivElement> {
    * https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/helpers/util.ts
    */
   getVerticalBreakpoint?: (height: number | null) => 'default' | 'sm' | 'md' | 'lg' | 'xl' | '2xl';
+  /** Banner component for the page */
+  banner?: React.ReactNode;
```
Comment (tlabaj) [inline packages/react-core/src/components/Page/Page.tsx:80]: So this is technically a new feature.

**Comment by tlabaj** [comment]
@lboehling @andrew-ronaldson I wonder if the design guidelines for Page should mention anything about banner since we now provide a prop.  I assume we want consumer to avoid placing banners below breadcrumbs.

**Review by wise-king-sullyman — APPROVED** [review]
Agree with the comment about expanding the prop description, but react code wise this looks good

**Review by kmcfaul — APPROVED** [review]
lgtm with the updated description

**Pull request merged**